v2/cli/term/event_page.go

```diff
@@ -0,0 +1,344 @@
+	e.workerInfo.SetBorderColor(workerPhaseColor).SetTitleColor(workerPhaseColor)
+	image := "DEFAULT"
+	if event.Worker.Spec.Container != nil {
+		image = event.Worker.Spec.Container.Image
+	}
+	infoText := fmt.Sprintf(
+		`[grey]Image: [white]%s
+[grey]Started: [white]%s
+[grey]Ended: [white]%s`,
+		image,
+		formatDateTimeToString(event.Worker.Status.Started),
+		formatDateTimeToString(event.Worker.Status.Ended),
+	)
+	if event.Worker.Status.Started != nil && event.Worker.Status.Ended != nil {
+		infoText = fmt.Sprintf(
+			"%s\n[grey]Duration: [white]%s",
+			infoText,
+			event.Worker.Status.Ended.Sub(*event.Worker.Status.Started),
+		)
+	}
+	infoText = fmt.Sprintf(
+		"%s\n[grey]Phase: %s%s",
+		infoText,
+		getTextColorFromWorkerPhase(event.Worker.Status.Phase),
+		event.Worker.Status.Phase,
+	)
+	e.workerInfo.SetText(infoText)
+}
+
+func (e *eventPage) fillJobsTable(event core.Event) {
+	const (
+		statusCol int = iota
+		nameCol
+		imageCol
+		ageCol
+		startedCol
+		endedCol
+		durationCol
+	)
+	e.jobsTable.Clear()
+	e.jobsTable.SetCell(
+		0,
+		statusCol,
+		&tview.TableCell{
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0,
+		nameCol,
+		&tview.TableCell{
+			Text:  "Name",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0, imageCol,
+		&tview.TableCell{
+			Text:  "Primary Image",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0,
+		ageCol,
+		&tview.TableCell{
+			Text:  "Age",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0,
+		startedCol,
+		&tview.TableCell{
+			Text:  "Started",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0,
+		endedCol,
+		&tview.TableCell{
+			Text:  "Ended",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	).SetCell(
+		0,
+		durationCol,
+		&tview.TableCell{
+			Text:  "Duration",
+			Align: tview.AlignCenter,
+			Color: tcell.ColorYellow,
+		},
+	)
+	for r, job := range event.Worker.Jobs {
+		row := r + 1
+		color := getColorFromJobPhase(job.Status.Phase)
+		created := ""
+		if job.Created != nil { // Older jobs didn't have this field
+			created = duration.ShortHumanDuration(
+				time.Since(*job.Created).Truncate(time.Second),
+			)
+		}
+		e.jobsTable.SetCell(
+			row,
+			statusCol,
+			&tview.TableCell{
+				Text:  getIconFromJobPhase(job.Status.Phase),
+				Align: tview.AlignLeft,
+				Color: color,
+			},
+		).SetCell(
+			row,
+			nameCol,
+			&tview.TableCell{
+				Text:  job.Name,
+				Align: tview.AlignLeft,
+				Color: color,
+			},
+		).SetCell(
+			row,
+			imageCol,
+			&tview.TableCell{
+				Text:  job.Spec.PrimaryContainer.Image,
+				Align: tview.AlignLeft,
+				Color: color,
+			},
+		).SetCell(
+			row,
+			ageCol,
+			&tview.TableCell{
+				Text:  created,
+				Align: tview.AlignLeft,
+				Color: color,
+			},
+		)
+		// TODO: Add age-- needs Job to track create time
```

Comment:
Is this still a TODO?  I see the created time added to the ageCol above.

Reply:
Nice catch. This can be addressed now. Will do so.